src/spikeinterface/sorters/launcher.py

```diff
@@ -221,6 +344,13 @@ def run_sorters(
     results : dict
         The output is nested dict[(rec_name, sorter_name)] of SortingExtractor.
     """
+
+    warnings.warn(
+        "run_sorters()is deprecated please use run_sorter_jobs() instead",
```

Comment:
Maybe adding the version here when this will be deprecated?

Reply:
done